platform/podjs/README.md

```diff
@@ -29,12 +29,102 @@ soon.
 
 Simply run `build.js` in the repository root.
 
-## Usage
+## How to setup a feature to build with podjs
 
-You will need to use this module for creating a polyPod-compatible
-feature. Follow these instructions:
+To create a polyPod compatible feature, you will need to use this module.
+Follow these instructions to do so:
 
-1. Add _podjs_ as a development dependency to your feature.
-2. Copy or link `node_modules/@polypoly-eu/podjs/dist/pod.js`.
-3. Include `pod.js` from your feature's main document - before the
-   feature's own code.
+1.  In order to point to `podjs` from the polypoly-eu repository,
+    add as `development dependencies` on your `package.json` the following:
+
+      <code>
+
+         "devDependencies": {
+            "@polypoly-eu/podjs": "file:../../platform/podjs",
```

Comment:
alt: change `../../platform` to `dummy/path/to/` Devs might or might not have features in the same directory as we do now.